README.md

```diff
@@ -77,20 +77,12 @@ Here's a rough sketch of how  I imagine the milestones of this project:
 
 ## Become a contributor
 
-### Contribute to course material
-If you are interested in contributing to course development, please fill the following form: https://danielkantor196881.typeform.com/to/V00Paz. The project is in a very early stage right now, so you might not be able to contribute right away. Your work will be released in this GitHub repository and you will show up as a contributor here.
+We welcome contributions for course material as well development contributions.In order to know more regarding your contributions to our project please checkout the [contributing guide](https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/contribution_details/CONTRIBUTING_GUIDE.md)
```

Comment:
> We welcome contributions for course material as well **as** development contributions

Reply:
> ns.In

Missing space here

I recommend running this through some spell checker that makes spotting these mistakes easy. Grammarly is one option that runs in the browser. There might be some plugin for your code editor

Reply:
https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/contribution_details/CONTRIBUTING_GUIDE.md :arrow_backward: that link doesn't seem right! It's linking to your fork. Perhaps the best would be to use a [relative link](https://github.blog/2013-01-31-relative-links-in-markup-files/)?

Reply:
(Which makes me realize, that I also did the mistake of specifically linking my fork instead of using relative links)

Reply:
> https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/contribution_details/CONTRIBUTING_GUIDE.md ◀️ that link doesn't seem right! It's linking to your fork. Perhaps the best would be to use a [relative link](https://github.blog/2013-01-31-relative-links-in-markup-files/)?

Can you please tell me about this @kantord.I'm unable to link this file.

Reply:
To learn more about relative links, consult the documentation I linked

Reply:
Or is it wrong/confusing for some reason?

Reply:
@kantord when I tried doing the same it showed `404` error

Reply:
hmm, what is the link that you tried that gave 404?

Reply:
Can you please tell me how do I link this `contribution guide` file with relative link?